java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TestJobIdGenerator.java

```diff
@@ -0,0 +1,49 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.dao.JobIdGenerator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class TestJobIdGenerator {
+    public HashSet<Integer> jobIdSet = new HashSet<>();
```

Comment:
It seems that Java HashSet is not thread safe. The check with `jobIdSet` may be incorrect.

Reply:
You're right. Maybe it doesn't matter. Because we can promise get each Job Id unique. But I will use hash table instead  of hashset.